src/main/java/org/twilmes/sql/gremlin/schema/TableUtil.java

```diff
@@ -61,11 +61,23 @@ public static Object convertType(Object value, TableColumn column) {
             case "string":
                 return value;
             case "integer":
-                return ((Number) value).intValue();
+                try {
```

Comment:
I wonder if this is correct way of converting these types, but it works for me so far, same to the rest of changes